solidity/contracts/l2/L1BitcoinDepositor.sol

```diff
@@ -13,7 +13,7 @@
 //               ▐████▌    ▐████▌
 //               ▐████▌    ▐████▌
 
-pragma solidity ^0.8.17;
+pragma solidity 0.8.17;
```

Comment:
Shouldn't this change be applied to other files, like `tbtc-v2/solidity/contracts/l2/L2WormholeGateway.sol`?

Reply:
Let's leave as-is for now. The other contracts are already deployed anyway.